config/__init__.py

```diff
@@ -5,25 +5,17 @@
 
 match FLASK_ENV:
     case "development":
-        from config.environments.development import (
-            DevelopmentConfig as Config,
-        )
+        from config.envs.development import DevelopmentConfig as Config
```

Comment:
love this

Reply:
So important I should have made it the title of this PR...